src/libs/actions/Task.ts

```diff
@@ -650,6 +650,27 @@ function setAssigneeChatReport(chatReport: OnyxTypes.Report) {
     Onyx.merge(ONYXKEYS.TASK, {assigneeChatReport: chatReport});
 }
 
+function setNewOptimisticAssignee(assigneeLogin: string, assigneeAccountID: number | undefined = undefined) {
```

Comment:
```suggestion
function setNewOptimisticAssignee(assigneeLogin: string, assigneeAccountID: number) { 
```
NAB let's make the prop required and generate the account id before calling this function

Reply:
@s77rt I updated.